src/sensitivity_interface.jl

```diff
@@ -14,9 +14,9 @@ function _adjoint_sensitivities(sol,sensealg,alg,g,t=nothing,dg=nothing;
                                    kwargs...)
 
   if sol.prob isa ODEProblem
-    adj_prob = ODEAdjointProblem(sol,sensealg,g,t,dg,checkpoints=checkpoints,
+    adj_prob = ODEAdjointProblem(sol,sensealg,g,t,dg; checkpoints=checkpoints,
                                  callback = callback,
-                                 abstol=abstol,reltol=reltol)
+                                 abstol=abstol,reltol=reltol, kwargs...)

```

Comment:
passing `kwargs` to the `ODEAdjointProblem` seems to be necessary to force the stepping also in the checkpointed interpolating adjoint. Otherwise, `tstops` that do not save anything are dropped.